src/openqaoa-core/algorithms/workflow_properties.py

```diff
@@ -262,6 +261,71 @@ def __init__(
     #     self._cvar_alpha = value
 
 
+class ErrorMitigationProperties(WorkflowProperties):
+    """
+    Optional, choose an error mitigation technique for the QAOA circuit. Currently supports only SPAM Twirling.
+
+    Parameters
+    ----------
+    error_mitigation_technique: str
+        The name of the error mitigation technique. Only 'spam_twirling' supported for now.
+    n_batches: Optional[int] = int
+        Number of batches in which the total number of shots is divided to. For every batch, we choose a set of qubits at random to which we apply X gates and classical negating. The dafault value is set to 10 to be comparable with most problem sizes in NISQ without creating too much of an overhead.
+    calibration_data_location: str
+        The path to the file containing calibration data for the specific device.
+    """
+
+    def __init__(
+        self,
+        error_mitigation_technique=None,
+        n_batches=10,
+        calibration_data_location=None,
+    ):
+        self.error_mitigation_technique = (
```

Comment:
@Q-lds this is the lower case

Reply:
Got it, thanks!

I would still force the datatype to be a string in the `init`, and then just do `.lower()`